TestServer/TestServerController.php

```diff
@@ -67,18 +67,6 @@ function __construct(LoggerInterface $logger, string $host_url, string $authProt
      */
     private function initTiqrLibrary()
```

Comment:
Now that this method is empty, can it be removed?

Reply:
🗑️